presets.go

```diff
@@ -14,6 +14,9 @@ type QueryPreset struct {
 }
 
 var PRESETS = map[string]QueryPreset{
+	"panama": QueryPreset{
+		include: []string{"panama", "panamá", "panama+city", "tocumen", "david"},
```

Comment:
```suggestion
		include: []string{"panama", "panamá", "tocumen"},
```
including `panama` means `panama city` is redundant. Also, `david` sounds too generic -- it can match lots of locations outside of Panama. Is there something better that can be used?

Reply:
You're right. I think it's ok to go without `david`